src/shared/condition.c

```diff
@@ -959,6 +969,7 @@ static const char* const assert_type_table[_CONDITION_TYPE_MAX] = {
         [CONDITION_CPUS] = "AssertCPUs",
         [CONDITION_MEMORY] = "AssertMemory",
         [CONDITION_ENVIRONMENT] = "AssertEnvironment",
+        [CONDITION_CPUFEATURE] = "AssertCPUFeature",
```

Comment:
I think `CONDITION_CPU_FEATURE` would be more in line with the other naming, i.e. place an underscore between ech word